test/unit/tasks/impersonation_admin_user_test.rb

```diff
@@ -19,7 +19,7 @@ class Tasks::ImpersonationAdminUserTest < ActiveSupport::TestCase
     users = User.where(username: 'example-username')
     assert_equal 0, User.where(username: '3scaleadmin').count
     assert_equal 3, users.count
-    assert_equal 2, User.where('username <> \'3scaleadmin\'').where('username <> \'example-username\'').count
+    assert_equal 3, User.where('username <> \'3scaleadmin\'').where('username <> \'example-username\'').count
```

Comment:
why it changed from `2` to `3` ?

Reply:
because in the Oracle tests it had the `master_account` in the DB at this point sometimes, so this way I add the `master_account` to the DB **always** and no just sometimes, so it is also 3 now 😄

Reply:
make sense 🍟